tests/Avalonia.Controls.UnitTests/ListBoxTests.cs

```diff
@@ -170,6 +171,109 @@ public void ScrollViewer_Should_Have_Correct_Extent_And_Viewport()
             Assert.Equal(new Size(100, 10), target.Scroll.Viewport);
         }
 
+        [Theory]
+        [InlineData(ItemVirtualizationMode.Simple)]
+        [InlineData(ItemVirtualizationMode.None)]
+        public void When_Added_Removed_AfterItems_Reset_Should_Work(ItemVirtualizationMode virtMode)
+        {
+            using (UnitTestApplication.Start(TestServices.StyledWindow))
+            {
+                var items = new ObservableCollection<string>();
+
+                Action create = () =>
```

Comment:
Could you turn these lambdas into local functions? It's a bit easier to read for me that way.

Reply:
@grokys no more lambdas now